AudioSystem/Editor/AudioClipContextMenuExtensions.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+namespace DUCK.AudioSystem.Editor
+{
+	public class AudioClipContextMenuExtensions : UnityEditor.Editor
```

Comment:
Utility for creating AudioConfig objects directly from selected AudioClip assets in the asset inspector

Reply:
Looks Fancy™